htdocs/api/playlist/appendfiletoplaylist.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Could you rename the file to camelCase starting with a lower case, so it’s consistent with the other code?